components/encumbered/vlc/Makefile

```diff
@@ -38,6 +39,8 @@ include $(WS_MAKE_RULES)/common.mk
 
 PATH= $(PATH.gnu)
 
+CONFIGURE_ENV += BUILDCC="/usr/bin/gcc"
```

Comment:
Is this the compiler used to build this component?  If so, then `/usr/bin/gcc` could point to any version.  Wouldn't be `$(CC)` or so better?